docs/java-rest/high-level/ml/put-calendar-job.asciidoc

```diff
@@ -5,14 +5,14 @@
 --
 [role="xpack"]
 [id="{upid}-{api}"]
-=== Put {anomaly-jobs} in calendar API
+=== Create {anomaly-jobs} in calendar API
```

Comment:
Aligning this with the REST API... you're not creating something, you're adding something to the calendar
```suggestion
=== Add {anomaly-jobs} to calendar API
```